package.json

```diff
@@ -31,7 +31,7 @@
     "author": "Moses Wejuli <mwejuli@companieshouse.gov.uk>",
     "license": "MIT",
     "dependencies": {
-        "@companieshouse/api-sdk-node": "^2.0.159",
+        "@companieshouse/api-sdk-node": "^2.0.162",
```

Comment:
the change needed is in https://github.com/companieshouse/api-sdk-node/releases/tag/2.0.163